ci/render.sh

```diff
@@ -22,6 +22,7 @@ do
   D2_LAYOUT=tala d2 src/examples/$ex/syntax/d2 src/examples/$ex/render/d2_tala.svg
   D2_LAYOUT=dagre d2 src/examples/$ex/syntax/d2 src/examples/$ex/render/d2_dagre.svg
   D2_LAYOUT=elk d2 src/examples/$ex/syntax/d2 src/examples/$ex/render/d2_elk.svg
+  # v9.2.2
   mmdc -i src/examples/$ex/syntax/mermaid -o src/examples/$ex/render/mermaid_dagre.svg
```

Comment:
By the way, if you have [`npx`](https://docs.npmjs.com/cli/v9/commands/npx) installed on your computer (it's usually included with new-ish versions of Node.JS and npm v7+), it might be better to do:

```suggestion
  npx --yes --package @mermaid-js/mermaid-cli@9.2.2 -- \
    mmdc -i src/examples/$ex/syntax/mermaid -o src/examples/$ex/render/mermaid_dagre.svg
```

This will automatically install `@mermaid-js/mermaid-cli@9.2.2` into a NPM cache folder and run it.

You can also replace the `9.2.2` with `latest`, and it should auto-update with the latest version, but generally for CI, you'd want to manually control versions.

If you leave the version blank (e.g. `@mermaid-js/mermaid-cli`), it should auto-pick the version from your `package.json` file if you stick `@mermaid-js/mermaid-cli` in your `dependencies`/`devDependencies`. This is usually the best way of doing things, because then you can use a program like [GitHub's Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#enabling-github-dependabot-version-updates) to automatically give you updates.

Reply:
Thank you for pointing this out @aloisklink 

I changed it to use npx in #33